test/e2e/disability-benefits-helpers.js

```diff
@@ -17,14 +18,39 @@ function completeEvidenceTypeInformation(client, data) {
 }
 
 function completeVAFacilitiesInformation(client, data) {
-  client
-    .selectDropdown('root_treatments_0_treatment_startTreatmentMonth', data.treatments[0].startTreatmentMonth)
-    .selectDropdown('root_treatments_0_treatment_startTreatmentDay', data.treatments[0].startTreatmentDay)
-    .fill('input[name="root_treatments_0_treatment_startTreatmentYear"]', data.treatments[0].startTreatmentYear)
-    .selectDropdown('root_treatments_0_treatment_endTreatmentMonth', data.treatments[0].endTreatmentMonth)
-    .selectDropdown('root_treatments_0_treatment_endTreatmentDay', data.treatments[0].endTreatmentDay)
-    .fill('input[name="root_treatments_0_treatment_endTreatmentYear"]', data.treatments[0].endTreatmentYear)
-    .fill('input[name="root_treatments_0_treatment_treatmentCenterName"]', data.treatments[0].treatmentCenterName);
+  for (let i = 0; i < 2; i++) {
```

Comment:
Can we do `data.treatments.forEach((treatment, index) => {...})`?

Reply:
Much better, changed.